microsetta_interface/templates/reports.jinja2

```diff
@@ -90,9 +90,23 @@
                 <div class="barcode-col barcode-text">
                     {{ sample.sample_barcode }}
                 </div>
-                <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
-                    <span class="sample-label">{{ _('Not Received Yet') }}</span>
-                </div>
+                {% if sample.latest_scan_timestamp is none %}
```

Comment:
I think there are a couple of variable names that need to be adjusted here. In [Sample.to_api()](https://github.com/ayobi/microsetta-private-api/blob/master/microsetta_private_api/model/sample.py#L58) in microsetta-private-api, you added `sample_latest_scan_timestamp` and `sample_last_update` but this block of code is expecting `latest_scan_timestamp` and `last_update`.